Calculator/OperatorTests/OperatorTests.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  OperatorTests.swift
+//  OperatorTests
+//
+//  Created by jiye Yi on 2023/01/27.
+//
+
+import XCTest
+@testable import Calculator
+final class OperatorTests: XCTestCase {
+
+    override func setUpWithError() throws {
+       
+    }
+
+    override func tearDownWithError() throws {
+    
+    }
+
+    func test_add_를실행하면_두개의값을더한값이_반환된다() throws {
+        //given
+        let lhs: Double = 12.5
+        let rhs: Double = 12.5
+        let expectation: Double = 25
+
+        //when
+        let result = try Operator.add.calculate(lhs: lhs, rhs: rhs)
+
+        //then
+        XCTAssertEqual(result, expectation)
+    }
+
+    func test_subtract_를실행하면_첫번째값에서_두번째값을뺀값이_반환된다() throws {
+        //given
+        let lhs: Double = 25.0
+        let rhs: Double = 12.5
+        let expectation: Double = 12.5
+
+        //when
+        let result = try Operator.subtract.calculate(lhs: lhs, rhs: rhs)
+
+        //then
+        XCTAssertEqual(result, expectation)
+    }
+   
+    func test_divide_를실행하면_첫번째값에서_두번째값을나눈값이_반환된다() throws {
+        //given
+        let lhs: Double = 25.0
+        let rhs: Double = 5.0
+        let expectation: Double = 5
+
+        //when
+        let result = try Operator.divide.calculate(lhs: lhs, rhs: rhs)
+
+        //then
+        XCTAssertEqual(result, expectation)
+    }
+    
+    func test_divide_두번째값이_0이라면_오류를던진다() throws {
+        //given
+        let lhs: Double = 5.0
+        let rhs: Double = 0
+        
+        //then
+        XCTAssertThrowsError(try Operator.divide.calculate(lhs: lhs, rhs: rhs)) { error in XCTAssertEqual(error as! CalculatorError, CalculatorError.divideError) }
```

Comment:
개행을 넣어주면 보기 깔끔할 것 같아요. 그리고 강제 캐스팅(`!`)보다는 옵셔널 캐스팅을 사용해주세요!
```suggestion
        XCTAssertThrowsError(try Operator.divide.calculate(lhs: lhs, rhs: rhs)) { error in 
            XCTAssertEqual(error as? CalculatorError, CalculatorError.divideError) 
        }
```